src/pre_commit_hooks/check_template_version.py

```diff
@@ -142,4 +136,4 @@ def main(argv: Union[Sequence[str], None] = None) -> int:
 
 
 if __name__ == "__main__":
-    SystemExit(main())
+    raise SystemExit(main())
```

Comment:
I'm pretty sure that it will be fine to add `raise` here, but I do want to make sure that it doesn't cause any confusion for pre-commit. Just want to make sure that the output of this pre-commit hook is either "Pass" or an info message printed to the console. And want to avoid blocking someone from committing code.

Reply:
I was also nervous about doing this, but I only did it to calm pylint down. I've just added a disable statement, so we can be less scared.